packages/components/src/Chip/Chip.tsx

```diff
@@ -16,11 +16,12 @@ export const Chip = ({
   invalid,
   label,
   value,
+  dataTestID,
   onClick,
   onKeyDown,
   children,
   role = "button",
-  tabIndex = 0,
+  tabIndex,
```

Comment:
https://github.com/GetJobber/atlantis/pull/1694/files#r1430475459 

> The original Chip implementation didn't do this, so we shouldn't either.

Reply:
I wanted to mention but I don't know if we really want to support this / if this needs to be fixed here but I tested and the Chip is not focusable through the keyboard if no `onClick` is provided but an `onKeyDown` is.

If you want to test for yourself remove `tabIndex` from the default args and only provide an `onKeyDown` but no `onClick` in storybook.

Reply:
Continuing this thread. If no `onClick` or `tabIndex` is provided then it isn't possible to view the tooltip for truncated labels.

Maybe @scotttjob has more context (although this was months ago) on this, based on [this](https://github.com/GetJobber/atlantis/pull/1694/files#r1430476308) comment there were issues with keyboard navigation setting the tab index to 0.

Again, not sure how likely this is but it is something to consider.
CC: @chris-at-jobber